web/packages/teleport/src/SessionRecordings/view/Xterm/Xterm.tsx

```diff
@@ -57,6 +64,10 @@ export default function Xterm({ tty }: { tty: Tty }) {
     term.open();
     term.term.focus();
 
+    term.registerCustomKeyEventHandler(
+      (e: KeyboardEvent) => !doNotBlockKeys.includes(e.key)
```

Comment:
What is this for?

Reply:
The session timeline will have keyboard shortcuts, but xterm is in focus, so it traps all the keyboard events

Reply:
Let's add this information to the comment on `doNotBlockKeys`.

```
// These keys are used for keyboard shortcuts in the session timeline,
// so we need to make sure that xterm.js doesn't trap the keyboard events.
const doNotBlockKeys = ['?', 't', 'h', 's', 'Escape'];
```